docker-compose.yml

```diff
@@ -9,6 +9,7 @@ web:
   environment:
     PG_USERNAME: postgres
     PG_HOST: db
+  command: bundle exec unicorn -l 0.0.0.0:3000
```

Comment:
This wasn't my experience; I was still getting WEBrick on startup until I changed this line.